docs/user-guide/ze-install-uninstall-ze.md

```diff
@@ -0,0 +1,48 @@
+# Uninstalling Zowe Explorer
+
+You can uninstall Zowe™ Explorer from Visual Studio Code (VS Code) if you no longer need to use it.
+
+:::info Required roles: systems administrator
+:::
+
+:::warning Important
+The uninstall process does not delete the profiles and credentials that you created when using the product from your computer. To delete the profiles from your computer, delete them before you uninstall Zowe Explorer. **Copied this admonition from the Zowe CLI uninstall doc. Can we use this for ZE too?**
```

Comment:
@adam-wolfe please respond

Reply:
Yes. this info is true for ZE as well. Although I wonder if users might be curious about how to clear their credentials from secure storage. Maybe a discussion for CLI/ZE standup.

Reply:
addressed per ZE squad discussion